src/rust/ide/view/src/documentation.rs

```diff
@@ -86,7 +70,7 @@ impl Model {
         let display_object = display::object::Instance::new(&logger);
         let div            = web::create_div();
         let dom            = DomSymbol::new(&div);
-        let size           = Rc::new(Cell::new(Vector2(VIEW_WIDTH,VIEW_HEIGHT)));
+        let size           = Rc::new(Cell::new(Vector2(VIEW_WIDTH-PADDING,VIEW_HEIGHT-PADDING-5.0)));
```

Comment:
This is using an inline constant. Either it should be a properly named constant or this should also be done in CSS.

Reply:
This is still here.